docs/products/postgresql/howto/monitor-database-with-datadog.rst

```diff
@@ -45,6 +45,9 @@ Using the ``avn service integration-list`` :ref:`Aiven CLI command <avn_service_
     "datadog_dbm_enabled": true
     }
     
+.. Note::
+    After completed the above steps successfully, Datadog Database Monitoring setup is ready.  These instructions replaced the steps in `Datadog Database Monitoring setup for PostgreSQL® <https://docs.datadoghq.com/database_monitoring/setup_postgres/>`_.
```

Comment:
> After completed the above steps successfully, Datadog Database Monitoring setup is ready.

``Executing the steps successfully results in enabling Datadog Database Monitoring for your service.``

> These instructions replaced the steps in `Datadog Database Monitoring setup for PostgreSQL® <https://docs.datadoghq.com/database_monitoring/setup_postgres/>`_.

How about including this info at the beginning of the article (after the info paragraphs and before Prerequisites, for example?

Also, what this instruction is actually replacing? What steps in particular you mean? Have there been any Aiven-specific instructions at https://docs.datadoghq.com/database_monitoring/setup_postgres/?
If so, maybe rephrase into ``This article replaces the instruction on how to set up Datadog Database Monitoring for Aiven for PostgreSQL that used to be available in `Setting up PostgreSQL <https://docs.datadoghq.com/database_monitoring/setup_postgres/>`_.``

Reply:
updated as per your suggestion, thanks!